#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>
#include <QList>
#include "staff.h"
#include "stafflist.h"

namespace Ui {
class MainWindow;
}

class MainWindow : public QMainWindow
{
    Q_OBJECT
    
public:
    explicit MainWindow(QWidget *parent = 0);
    ~MainWindow();
    
private slots:
    void process();
    void addDynamicProperty();
    void writeToFile();

private:
    Ui::MainWindow *ui;
    StaffList list;
};

#endif // MAINWINDOW_H
